foxy/distribution.yaml

```diff
@@ -2673,6 +2673,12 @@ repositories:
       url: https://github.com/ros2/rcl_logging.git
       version: foxy
     status: maintained
+  rclada:
+    doc:
+      type: git
+      url: https://github.com/ada-ros/ada4ros2.git
+      version: foxy    
```

Comment:
There's trailing whitespace at the end of this line, which is why CI is failing.